samples/RaspberryPiInstructions.md

```diff
@@ -1,70 +1,104 @@
 
-* If you get restore errors, make sure you have a nuget.config file next to your csproj that includes the dotnet-core myget feed: `<add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />`.
+Note: The pattern documented below is the same one we use for [Docker](https://github.com/dotnet/dotnet-docker). You can look at .NET Core Dockerfiles to learn more about specific installation needs.
 
-```
-<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-  <packageSources>
-    <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
-  </packageSources>
-</configuration>
+1. Download .NET Core via curl (choose the version that works for you):
+  * 2.2 ARM32: `curl -SL --output dotnet.tar.gz https://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.2.103/dotnet-sdk-2.2.103-linux-arm.tar.gz`
+  * 3.0 ARM32: `curl -SL --output dotnet.tar.gz https://dotnetcli.blob.core.windows.net/dotnet/Sdk/3.0.100-preview-010184/dotnet-sdk-3.0.100-preview-010184-linux-arm.tar.gz`
+  * 3.0 ARM64: `curl -SL --output dotnet.tar.gz https://dotnetcli.blob.core.windows.net/dotnet/Sdk/3.0.100-preview-010184/dotnet-sdk-3.0.100-preview-010184-linux-arm64.tar.gz`
+
+  Note: The latest download links are available at the [.NET Core Download page](https://dotnet.microsoft.com/download/archives).
+2. Unpack .NET Core to a global location:
+  * `sudo mkdir -p /usr/share/dotnet`
+  * `sudo tar -zxf dotnet.tar.gz -C /usr/share/dotnet`
+3. Symbolically link dotnet into a path location:
+  * `sudo ln -s /usr/share/dotnet/dotnet /usr/bin/dotnet`
+
+You should be able to run .NET Core at this point:
+
+```console
+dotnet
 ```
 
-## Publishing an app to run on the Pi:
+## Installing the .NET Core SDK on Raspberry Pi on Win10 IoT Core
 
-* Run `dotnet publish -r <runtime identifier>` for example `dotnet publish -r win-arm` to publish the application for windows and `dotnet publish -r linux-arm` for Linux running on Raspberry Pi.
+Install [Windows 10 IoT Core](https://docs.microsoft.com/en-us/windows/iot-core/getstarted) on your Pi.
 
-* Under `./bin/Debug/netcoreapp2.1/<runtime identifier>/publish` or `.\bin\Debug\netcoreapp2.1\<runtime identifier>\publish` you will see the whole self contained app that you need to copy to your Raspberry Pi.
+Install the [.NET Core 3.0 SDK for Windows ARM32](https://dotnet.microsoft.com/download/dotnet-core/3.0).
 
+## Creating and Running an Application on the Pi
 
-## Getting the app to run on the Pi.
+It is easy to run an application on the Pi. The following commands are for Linux, but switching the casing will enable the same commands to work on Windows.
 
-### Linux
+Console App:
 
-* Install [Linux](https://www.raspberrypi.org/downloads/) on your Pi.
+1. `dotnet new console -o app` // Generate app from template
+2. `cd app`
+3. `dotnet run` // Build and run app
+4. `./bin/Debug/netcoreapp3.0/app` // Alternatively, run app directly
```

Comment:
Shouldn't this be `dotnet /bin/Debug/netcoreapp3.0/app.dll? When I try to run directly I get 

```
leecow at Brandywine in ~/test
$ ./bin/Debug/netcoreapp2.2/test
-bash: ./bin/Debug/netcoreapp2.2/test: No such file or directory
```

Reply:
This pattern only works with 3.0